udata/core/user/models.py

```diff
@@ -120,9 +122,14 @@ def external_url(self):
         return self.url_for(_external=True)
 
     @property
-    def visible(self):
+    def visible_for_search(self):
         count = self.metrics.get('datasets', 0) + self.metrics.get('reuses', 0)
-        return count > 0
+        return count > 0 and self.active
+
+    @property
+    def visible(self):
+        return self.active or (
+            not current_user.is_anonymous and current_user.sysadmin)
```

Comment:
1. I think this only correct in views because it needs a `current_user`. By example, [here](https://github.com/opendatateam/udata/blob/dev/udata/core/user/search.py#L100) it will always be `False`).
2. no, I think model properties should not involve `current_user` which is only views related
To simplify, the clause should only be `return self.active` and we will ad more rules later if needed. The `current_user` should only happen in views (ie. `user.visible and current_user...`)